src/entitysdk/downloaders/ion_channel_model.py

```diff
@@ -0,0 +1,22 @@
+"""Download functions for IonChannelModel entities."""
+
+from pathlib import Path
+
+
+def download_one_mechanism(client, ic, mechanisms_dir="./mechanisms"):
+    """Download one mechanism file.
+
+    Args:
+        client (Client): EntitySDK client
+        ic (IonChannelModel): IonChannelModel entitysdk object
+        mechanisms_dir (str or Pathlib.Path): directory to save the mechanism file
+    """
+    mechanisms_dir = Path(mechanisms_dir)
+    mechanisms_dir.mkdir(parents=True, exist_ok=True)
+    asset = client.download_assets(
```

Comment:
Why just one? Aren't there multiple neuron-mod assets?

Reply:
Each mod file is in its own IonChannelModel entity, so we have to get them one at a time